coordinator/src/services/translation/practitioner.ts

```diff
@@ -46,14 +46,41 @@ export function convertResponsibleParty(
 
 function convertPractitionerRole(fhirBundle: fhir.Bundle, fhirPractitionerRole: fhir.PractitionerRole): peoplePlaces.AgentPerson {
   const fhirPractitioner = resolveReference(fhirBundle, fhirPractitionerRole.practitioner)
-  const hl7V3AgentPerson = createAgentPerson(fhirBundle, fhirPractitionerRole, fhirPractitioner)
-  const fhirOrganization = resolveReference(fhirBundle, fhirPractitionerRole.organization)
-  hl7V3AgentPerson.representedOrganization = convertOrganizationAndProviderLicense(fhirBundle, fhirOrganization)
+  const hl7V3AgentPerson = createAgentPerson(fhirPractitionerRole, fhirPractitioner)
+  const fhirOrganization = resolveOrganizationReference(fhirBundle, fhirPractitionerRole.organization)
+  let fhirHealthcareService: fhir.HealthcareService
+  if (fhirPractitionerRole.healthcareService) {
+    fhirHealthcareService = resolveReference<fhir.HealthcareService>(fhirBundle, fhirPractitionerRole.healthcareService[0])
+  }
+  hl7V3AgentPerson.representedOrganization = convertOrganizationAndProviderLicense(fhirBundle, fhirOrganization, fhirHealthcareService)
   return hl7V3AgentPerson
 }
 
+function isReference(reference: fhir.PractitionerRoleOrganization): reference is fhir.Reference<fhir.Organization> {
+  return (reference as fhir.Reference<fhir.Organization>).reference !== undefined
+}
+
+function resolveOrganizationReference(fhirBundle: fhir.Bundle, practitionerRoleOrganization: fhir.PractitionerRoleOrganization): fhir.Organization {
+  if (isReference(practitionerRoleOrganization)) {
+    return resolveReference(fhirBundle, practitionerRoleOrganization)
+  }
+  return createMiniumumOrganization(practitionerRoleOrganization.display, practitionerRoleOrganization.identifier)
+}
+
+function createMiniumumOrganization(display: string, identifier: Array<fhir.Identifier>){
+  return {
+    name: display,
+    identifier,
+    type: [{
+      coding: [{
+        system: "https://fhir.nhs.uk/R4/CodeSystem/organisation-role",
+        code: "RO197"
```

Comment:
I've had a think about this and I think it's a bad idea to set the type. Any Organization can be referred to by its identifier and that doesn't mean it's a trust. It's only in the particular case of PractitionerRole mappings that we want to treat it as such. If we start using this method in other places we're going to run into issues